examples/example-runner-cpu/src/main.rs

```diff
@@ -0,0 +1,54 @@
+use minifb::{Key, Window, WindowOptions};
+use rayon::prelude::*;
+use spirv_std::{vec2, Vec4};
+use std::time::Instant;
+
+fn color_u32_from_vec4(v: Vec4) -> u32 {
```

Comment:
Maybe a comment here about why it is OK to ignore sRGB here (or add a TODO about respecting sRGB) ?

Reply:
done